frontends/web/src/routes/device/waiting.tsx

```diff
@@ -14,71 +14,58 @@
-                        <div>
-                            <AppLogo />
-                            <div className="box large">
-                                <h3 className={style.waitingText}>{t('welcome.insertDevice')}</h3>
-                                <p className={style.waitingDescription}>{t('welcome.insertBitBox02')}</p>
-                            </div>
-                            {
-                                testing && (
-                                    <div className={style.testingContainer}>
-                                        <SkipForTesting show={!!testing} />
-                                    </div>
-                                )
-                            }
+    return (
+        <div className="contentWithGuide">
+            <div className="container">
+                <Header title={<h2>{t('welcome.title')}</h2>} />
+                <div className="content padded narrow isVerticallyCentered">
+                    <div>
+                        <AppLogo />
+                        <div className="box large">
+                            <h3 className={style.waitingText}>{t('welcome.insertDevice')}</h3>
+                            <p className={style.waitingDescription}>{t('welcome.insertBitBox02')}</p>
                         </div>
+                        {
+                            debug && (
+                                <div className={style.testingContainer}>
+                                    <SkipForTesting show />
+                                </div>
+                            )
+                        }
                     </div>
-                    <Footer>
-                        <SwissMadeOpenSource />
-                    </Footer>
                 </div>
-                <Guide>
-                    <Entry entry={t('guide.waiting.welcome')} shown={true} />
-                    <Entry entry={t('guide.waiting.getDevice')} />
-                    <Entry entry={t('guide.waiting.lostDevice')} />
-                    <Entry entry={t('guide.waiting.internet')} />
-                    <Entry entry={t('guide.waiting.deviceNotRecognized')} />
-                    <Entry entry={t('guide.waiting.useWithoutDevice')} />
-                </Guide>
+                <Footer>
+                    <SwissMadeOpenSource />
+                </Footer>
             </div>
-        );
-    }
+            <Guide>
+                <Entry entry={t('guide.waiting.welcome')} shown={true} />
+                <Entry entry={t('guide.waiting.getDevice')} />
+                <Entry entry={t('guide.waiting.lostDevice')} />
+                <Entry entry={t('guide.waiting.internet')} />
+                <Entry entry={t('guide.waiting.deviceNotRecognized')} />
+                <Entry entry={t('guide.waiting.useWithoutDevice')} />
+            </Guide>
+        </div>
+    )
 }
-
-const loadHOC = load<TestingProps, TranslateProps>(() => debug ? { testing: 'testing' } : {})(Waiting);
```

Comment:
Actually the behavior changes with this PR, before this PR:

- `debug` from 'utils/env.js' reads `process.env.NODE_ENV === 'development'`, that is set by `react-scripts`: running `npm start` it is `'development'`, while `npm test` it is set to `'test'` and `npm run build` builds with `'production'`, see https://create-react-app.dev/docs/adding-custom-environment-variables/
- if debug is true (should always be true in webdev) it makes a request to http://localhost:8082/api/testing
- backend: https://github.com/digitalbitbox/bitbox-wallet-app/blob/d83626fb086cdd71df97aba9d84e407ae4e42925/backend/backend.go#L407-L410
- with `make servewallet` the testing api returns `true`, and `make servewallet-mainnet` returns `false`

But with this PR the frontend now always runs with testing true in webdev mode.